source/includes/oai-pmh/_root.md

```diff
@@ -16,6 +16,52 @@ not available for download.
 
 `https://zenodo.org/oai2d`
 
+
+## First steps with Sickle
+
+In order to better understand the capabilities of OAI let’s do some harvesting using a tool called ```Sickle```. 
+Let's first install the ```Sickle``` package using ```pip```:
```

Comment:
```suggestion
Let's first install the `Sickle` package using `pip`:
```

Not needed for this PR (since this text will be a comment in the Python codeblock), but FYI, for "inline" monospace text single backticks work the same.